app/javascript/mastodon/features/compose/components/search.jsx

```diff
@@ -386,7 +387,11 @@ class Search extends PureComponent {
             </div>
           ) : (
             <div className='search__popout__menu__message'>
-              <FormattedMessage id='search_popout.full_text_search_disabled_message' defaultMessage='Not available on {domain}.' values={{ domain }} />
+              {searchEnabled ? (
```

Comment:
should this be `signedIn` instead of `searchEnabled`?

Reply:
You're right! Fixing that.